fe1-web/src/features/evoting/objects/__tests__/utils.ts

```diff
@@ -0,0 +1,157 @@
+);
+
+export const mockBallotOption1 = 'Ballot Option 1';
+export const mockBallotOption2 = 'Ballot Option 2';
+export const mockBallotOptions = [mockBallotOption1, mockBallotOption2];
+
+export const mockQuestionObject1: Question = {
+  id: mockQuestionId1.toString(),
+  question: mockQuestion1,
+  voting_method: STRINGS.election_method_Plurality,
+  ballot_options: mockBallotOptions,
+  write_in: false,
+};
+
+export const mockQuestionObject2: Question = {
+  id: mockQuestionId2.toString(),
+  question: mockQuestion2,
+  voting_method: STRINGS.election_method_Approval,
+  ballot_options: mockBallotOptions,
+  write_in: true,
+};
+
+export const mockQuestions: Question[] = [mockQuestionObject1, mockQuestionObject2];
+
+export const mockRegisteredVotes: RegisteredVote[] = [
+  {
+    createdAt: 0,
+    messageId: '0',
+    sender: 'sender 1',
+    votes: [
+      { id: 'id1', question: 'q1', vote: [0] },
+      { id: 'id2', question: 'q2', vote: [0] },
+    ],
+  },
+  {
+    createdAt: 1,
+    messageId: '1',
+    sender: 'sender 2',
+    votes: [
+      { id: 'id3', question: 'q3', vote: [0] },
+      { id: 'id4', question: 'q4', vote: [0] },
+    ],
+  },
+];
+
+export const mockElectionNotStarted = new Election({
+  lao: mockLaoIdHash,
+  id: mockElectionId,
+  name: mockElectionName,
+  version: VERSION,
+  createdAt: TIMESTAMP,
+  start: TIMESTAMP,
+  end: CLOSE_TIMESTAMP,
+  questions: mockQuestions,
+  electionStatus: ElectionStatus.NOT_STARTED,
+});
+
+export const mockElectionOpened = new Election({
+  lao: mockLaoIdHash,
+  id: mockElectionId,
+  name: mockElectionName,
+  version: VERSION,
+  createdAt: TIMESTAMP,
+  start: TIMESTAMP,
+  end: CLOSE_TIMESTAMP,
+  questions: mockQuestions,
+  electionStatus: ElectionStatus.OPENED,
+  registeredVotes: mockRegisteredVotes,
+});
+
+export const mockElectionTerminated = new Election({
+  lao: mockLaoIdHash,
+  id: mockElectionId,
+  name: mockElectionName,
+  version: VERSION,
+  createdAt: TIMESTAMP,
+  start: TIMESTAMP,
+  end: CLOSE_TIMESTAMP,
+  questions: mockQuestions,
+  electionStatus: ElectionStatus.TERMINATED,
+  registeredVotes: mockRegisteredVotes,
+});
+
+export const mockRegistedVotesHash = EndElection.computeRegisteredVotesHash(mockElectionOpened);
+
+export const mockVoteVotes1 = new Set([0]);
+export const mockVoteVotes2 = new Set([1, 0]);
+
+export const mockVoteId1 = CastVote.computeVoteId(mockElectionNotStarted, 0, mockVoteVotes1);
+export const mockVoteId2 = CastVote.computeVoteId(mockElectionNotStarted, 1, mockVoteVotes2);
+
+export const mockVote1: Vote = {
+  id: mockVoteId1.toString(),
+  question: mockQuestionId1.valueOf(),
+  vote: [0],
+};
+
+export const mockVote2: Vote = {
+  id: mockVoteId2.toString(),
+  question: mockQuestionId2.valueOf(),
+  vote: [1],
+};
+
+export const mockVotes = [mockVote1];
+
+export const mockElectionResultHash = Hash.fromStringArray(mockVoteId1.valueOf());
+
+// id: string; result: { ballot_option: string; count: number }[]
```

Comment:
```suggestion
// ElectionResult format : { id: string; result: [{ ballot_option: string; count: number }] }
```
But is this really useful ?